text/dotnet-core/rfcs/0004-extend-dotnet-sdk.md

```diff
@@ -0,0 +1,102 @@
+# Extend .Net SDK Dependency
+
+## Proposal
+
+Reconfigure the .Net SDK dependency to be identical to the one that can be downloaded from Microsoft which includes the runtimes that the SDK is dependent on to run.
+
+## Motivation
+As of [.Net SDK version ~6.0.100 publish no longer properly functions for
+certain types of .Net
+apps](https://github.com/paketo-buildpacks/dotnet-core/issues/670). Upon
+testing we were able to get the buildpack to function again if we used an .Net
+SDK dependency that still had the runtimes libraries packaged with it. After
+further investigation we discovered that the removal of those local runtimes
+was what was causing the failure, leading us to believe that the .Net SDK no
+longer follows the runtime search path that it used to.
+
+It is becoming ever more apparent to me that Microsoft is dead set on not
+having flexibility of file location during `dotnet publish` therefore I think
+that we should capitulate to Microsoft and create an expected .Net Hive
+installation, which is a layer where the file structure matches the layout that
```

Comment:
Some more details here would be appreciated, either the proposed file structure of the layer (compared to the current state), or an external link with similar information.